examples/vkcs-vpnaas-service-create/base.tf

```diff
@@ -0,0 +1,25 @@
+data "vkcs_networking_network" "extnet" {
+  name = "ext-net"
+}
+
+resource "vkcs_networking_network" "network" {
+	name           = "vpnaas_network"
+ 	admin_state_up = "true"
+}
+
+resource "vkcs_networking_subnet" "subnet" {
+ 		network_id = "${vkcs_networking_network.network.id}"
+ 		cidr       = "192.168.199.0/24"
+ 		ip_version = 4
+}
+
+resource "vkcs_networking_router" "router" {
+  name = "router"
+  admin_state_up = "true"
```

Comment:
i'm really not sure we need to include this attribute to examples

Reply:
Removed it from all examples